src/luaotfload-colors.lua

```diff
@@ -299,8 +299,7 @@ add_color_callback = function ( )
                         color_callback_name)
         add_to_callback("hpack_filter",
                         function (head, groupcode)
-                            if  groupcode == "hbox"          or
```

Comment:
This breaks in some edge cases because LuaTeX doesn't know how a box will be used when it is packed. E.g.

```
\setbox0{\somecolorfont abc} \box0
```
would no longer color `abc` if it is issued in vmode.